lib/bridge/IrcHandler.js

```diff
@@ -201,6 +207,55 @@ IrcHandler.prototype.onMessage = Promise.coroutine(function*(req, server, fromUs
         return;
     }
 
+    // For topics, set matrix topic as AS bot user
+    if (action.type === "topic") {
+        // Only bridge topics for rooms created by the bridge, via !join or an alias
+        let origins = ["join", "alias"];
+        let mappings = yield this.ircBridge.getStore().getMappingsForChannelByOrigin(
+            server, channel, origins
+        );
+        let promises = mappings.map((entry) => {
+
```

Comment:
Too many needless blank lines. Here, `:227`, `:233`, `:243`. I like blank lines when they are used to express "blocks" of code, much like how you write paragraphs. This isn't what is being done here though.